### DISCUSSION TITLE
What happened to `launchInComposition`?

### DISCUSSION THREAD

**Discussion by yektasarioglu** [Foso/Jetpack-Compose-Playground#35] [Q&A]
I'm using `1.0.0-alpha08` of Jetpack Compose. And I also examine other compose samples along the development to understand the concepts. One of the samples uses `launchInComposition()` to trigger some coroutine code. But I could not find `launchInComposition()` in `androidx.compose.runtime` package. Is this something deprecated already?

**Comment by Foso** [comment]
Hi, i haven't used it, but it looks like, it's now called LaunchedTask https://android-review.googlesource.com/c/platform/frameworks/support/+/1435674
